### DISCUSSION TITLE
Will the work with older Caddx / Interlogix NX8E systems?

### DISCUSSION THREAD

**Discussion by yombo** [taligentx/dscKeybusInterface#261] [dscKeybusInterface Discussion] [answered]
Saw the list of supported and unsupported hardware, just wondering if NX8E is supported? The Alula BAT-Connect works with so many variations of DSC/GE/Interlogix, that perhaps it might be possible it's already supported.

**Comment by taligentx** [comment, marked as answer]
Hi @yombo - thanks for the heads-up on the BAT-Connect, seems like an interesting product with the multi-system compatibility - it seems like it's been programmed with each separate manufacturer protocol.  For example, it supports DSC and Honeywell Vista, but these are totally different protocols.

The wiring is also different - DSC uses two data lines (green and yellow lines), while the BAT-Connect uses a single line (green) to connect to the Interlogix panel.  This means Interlogix is using its own protocol that is not compatible with DSC and currently wouldn't work with this library.  

The current `develop` branch is structured to accommodate different protocols, if someone decodes the Interlogix protocol it may be possible to add this and other manufacturer protocols to the library.  Honeywell Vista seems like a good future target as it's already decoded.

**Reply by yombo** [reply]
Oh, ok. Thanks for the response. I thought they shared the same heritage, so I figured it might work. I'll have to look for alternative solutions. :-)
